src/js/components/dashboard/filters/FiscalYearFilter.jsx

```diff
@@ -0,0 +1,96 @@
+/**
+ * FiscalYearFilter.jsx
+ * Created by Lizzie Salita 10/16/19
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import FiscalYear from './FiscalYear';
+
+const propTypes = {
+    selectedFY: PropTypes.array,
+    pickedFy: PropTypes.func,
+    allFy: PropTypes.array
+};
+
+export default class FiscalYearFilter extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.saveAllYears = this.saveAllYears.bind(this);
+    }
+
+    saveAllYears() {
+        if (this.props.selectedFY.length !== this.props.allFy.length) {
+            // Add years that are not yet selected
+            this.props.allFy.forEach((fy) => {
+                if (!this.props.selectedFY.includes(fy.year)) {
+                    this.props.pickedFy(fy.year);
+                }
+            });
+        }
+        else {
+            // Remove years that are still selected
+            this.props.allFy.forEach((fy) => {
+                if (this.props.selectedFY.includes(fy.year)) {
+                    this.props.pickedFy(fy.year);
+                }
+            });
+        }
+    }
+
+    render() {
+        let allFY = true;
+        const leftCount = Math.ceil(this.props.allFy.length / 2);
```

Comment:
Just bringing up the question created by the earlier comment, is it always going to be 2 columns or will it become 3 at a certain point? If it's staying at 2 then my earlier comment is probably also extraneous.

Reply:
I think it will stay two columns since we're modeling it after advanced search: 
<img width="309" alt="TimePeriod" src="https://user-images.githubusercontent.com/7108785/67305279-1931b980-f4c3-11e9-8cea-b0242eeb5245.png">

Reply:
Maybe it's the larger gap between them that makes it look like less of an empty space on the side. That makes sense though. In that case ignore the previous comment as well.